### PR TITLE
Added instructions to run Rally as a Kubernetes Job or a Docker container

### PR DESCRIPTION
**Pull request by gdmello**

### CODE REVIEW

File: docs/install.rst [hunk 1]
```diff
@@ -80,6 +80,138 @@ git
...
+    sudo apt-get install git
+
+
+**Red Hat / CentOS / Amazon Linux**
+
+::
+
+    sudo yum install git
+
+
```
Comment (danielmitterdorfer) [inline docs/install.rst]: This part is duplicated now in `install.rst` (maybe a merge problem?). Can you please remove the duplicate?

File: docs/install.rst [hunk 2]
```diff
@@ -132,6 +264,36 @@ Whenever you want to use Rally, run the activation script (step 2 above) first.
...
+        This documentation is for the version of Rally currently under development. We do not provide offline installation packages for development versions.
+        Were you looking for the `documentation of the latest stable version <//esrally.readthedocs.io/en/stable/>`_?
+
+If you are in a corporate environment where your servers do not have any access to the Internet, you can use Rally's offline installation package. Follow these steps to install Rally:
+
+1. Install all prerequisites as documented above.
+2. Download the offline installation package for the `latest release <https://github.com/elastic/rally/releases/latest>`_ and copy it to the target machine(s).
+3. Decompress the installation package with ``tar -xzf esrally-dist-*.tar.gz``.
+4. Run the install script with ``sudo ./esrally-dist-*/install.sh``.
+
```
Comment (danielmitterdorfer) [inline docs/install.rst]: Also the part up to here is duplicated.

File: docs/install.rst [hunk 3]
```diff
@@ -132,6 +264,36 @@ Whenever you want to use Rally, run the activation script (step 2 above) first.
...
+Kubernetes Job
+------------------
```
Comment (danielmitterdorfer) [inline docs/install.rst]: Restructured text is quite picky: the length of the underline must match the length of the text above.

**Comment by gdmello** [comment]
@danielmitterdorfer - I'm sorry for the delay, was on vacation. The issues were caused by a bad merge on my part, I'm sorry, I've fixed them all up, please re-review.

**Comment by danielmitterdorfer** [comment]
Thanks @gdmello and no worries on the delay. :) I've reviewed the changes: looks good now and the doc build also finishes fine. I'll merge it now. This will be live immediately in the [latest](esrally.readthedocs.io/en/latest/) (i.e. master) docs and be included in the next release of Rally (probably 0.8.2).

**Comment by gdmello** [comment]
Thanks @danielmitterdorfer! That's great!

**Pull request merged**